pkg/controllers/streaming/config.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package streaming
+
+const (
+	kustomizePrefix     = "riff-streaming"              // kustomize adds this prefix to all our resource names
+	providerImages      = kustomizePrefix + "-provider" // contains image names for the provider
+	liiklusImageKey     = "liiklus"
```

Comment:
Let's make these keys more obvious that they represent images. That will make detection with our scanner a bit easier.

```suggestion
	liiklusImageKey     = "liiklusImage"
```

Even more scanner friendly would be a configmap like:

```
data:
  liiklus: |
    image: ...
```

...and then marshal the liiklus string into a map to get the image value. I don't think that's worthwhile at this point, but it's how Knative approaches their configs.